robyn/robyn.pyi

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+from typing import Callable, Optional, Tuple
+
+class SocketHeld:
+    def __init__(self, url: str, port: int):
+        pass
+    def try_clone(self) -> SocketHeld:
+        pass
+
+class Server:
+    def __init__(self):
+        pass
+    def add_directory(
+        self,
+        route: str,
+        directory_path: str,
+        index_file: Optional[str],
+        show_files_listing: bool,
+    ):
+        pass
+    def add_header(self, key: str, value: str):
```

Comment:
Even if the method just returns None it's generally preferable to be explicit about that to avoid any potential differences in inference between type checkers.

Reply:
Got it! I will change it in the next release! :D